### ISSUE TITLE
[Arcs] Survival Override and "do as much as possible"

### ISSUE THREAD

**Issue by turmoilbyrd** [LederCards/cards#94]
There has been discussion about [Survival Overrides](https://cards.ledergames.com/card/ARCS-L18), whether it is possible to

1. destroy 1 Loyal ship to destroy 0 rival ship. Or
2. (!) destroy 0 Loyal ship to destroy 1 rival ship

Since destroy is defined with **return**, and return is governed by [Do as much as Possible](https://arcs.seiyria.com/#22.3.4-do-as-much-as-possible).

**Comment by turmoilbyrd** [comment]
My opinion is that both should be No

Reasoning:
> **Martyr (Move):** Destroy... to destroy...
Is a shorthand for
> When you spend a pip or a resource to **move**, you may instead (destroy... to destroy...)

So the "destroy to destroy" clause is put under a "may", this means [DAMAP](https://arcs.seiyria.com/#22.3.4-do-as-much-as-possible) rule, which requires a "must", doesn't apply anymore.

And we can apply Guerric's reasoning about "X to Y" construct on the "destroy to destroy" clause
> "To" is a load-bearing word as the community has understood up to this point. If an action initiates with an "X to Y" structure you must be able to do Y to initiate the action and perform X
So both actions must be possible (and no DAMAP)

**Comment by jdyearsley** [comment]
You cannot. Broadly, the "X to Y" construct should be read as "To do Y, you must do X as a prerequisite."

**Comment by turmoilbyrd** [comment]
> You cannot. Broadly, the "X to Y" construct should be read as "To do Y, you must do X as a prerequisite." Do you feel this rises to the level of an FAQ entry?

So how about the other way around, can you destroy 1 Loyal ship to destroy 0 rival ship?

**Comment by jdyearsley** [comment]
Yes, I will bite this bullet. Thematically it's justified—normally you can't just make ships blow themselves up, but now they've got overrides installed.

**Comment by turmoilbyrd** [comment]
> Yes, I will bite this bullet. Thematically it's justified—normally you can't just make ships blow themselves up, but now they've got overrides installed. I'll add this as an FAQ.

Ok, if "X to Y" only works one way (you can do X even if you cannot do Y), this may mean Yes on most of this

- #31

**Comment by jdyearsley** [comment]
Adding retroactive logic like that is _incredibly_ thorny. It would have to exclude cases like [Gatekeepers](https://cards.ledergames.com/card/ARCS-CC03?q=gatekeepers)—can I discard it even if I can't place _all_ those ships? No, that would be silly and unintuitive. So at that point I would have to distinguish resolution of **Y** in cases of full, partial, and none.
